core/executor/Cargo.toml

```diff
@@ -8,19 +8,20 @@ error-chain = "0.12"
 parity-codec = { version = "2.0" }
 sr-io = { path = "../sr-io" }
 substrate-primitives = { path = "../primitives" }
+substrate-trie = { path = "../trie" }
 substrate-serializer = { path = "../serializer" }
 substrate-state-machine = { path = "../state-machine"  }
 sr-version = { path = "../sr-version" }
 serde = "1.0"
 serde_derive = "1.0"
 wasmi = "0.4"
 byteorder = "1.1"
-triehash = "0.2"
+trie-root = { git = "https://github.com/paritytech/trie" }
 twox-hash = "1.1.0"
 lazy_static = "1.0"
 parking_lot = "*"
 log = "0.3"
-hashdb = "0.2.1"
+hash-db = { git = "https://github.com/paritytech/trie" }
```

Comment:
kind of unfortunate that we are moving back off of crates.io deps and into GitHub land

Reply:
once it stablises (i.e. sometime between now and 1.0 final), i'm happy to publish and move back.